internal/config/config.go

```diff
@@ -1005,8 +1011,32 @@ func validateDuplicate(strSlice []string, sliceType string) error {
 	return nil
 }
 
+func validateLabelSelectorMatchExpressions(matchExpressions []metav1.LabelSelectorRequirement) error {
```

Comment:
Since this is used in the function below, can we put it after it?
Ideally, higher level functions should come first.

Reply:
done.